slack-server-integration-common/src/main/java/com/atlassian/plugins/slack/api/client/DefaultSlackClientProvider.java

```diff
@@ -42,6 +42,7 @@
     // if timeout is less than the time during which Slack tries to load an image, exception is thrown,
     // BackoffRetryInterceptor performs the request again, with the same result causing up to 8 duplicates
     private static final int DEFAULT_READ_TIMEOUT_MILLISECONDS = 15 * 1000;
+    private static final int DEFAULT_WRITE_TIMEOUT_MILLISECONDS = 120 * 1000;
```

Comment:
The PR description mentions 90 seconds timeout while here 120 seconds are set. Is it a typo in the code or in the description? I think both values are fare enough. I just wanted to attract your attention to the noticed inconsistency.

Reply:
🤦‍♂️

Reply:
I had put 90 seconds in the code at first and then change it to be more lenient but when I was writing the PR description I forgot about the change.
Thank you for bringing it out.